lib/shoryuken/environment_loader.rb

```diff
@@ -45,12 +45,16 @@ def config_file_options
       fail ArgumentError, "The supplied config file #{path} does not exist" unless File.exist?(path)
 
       if (result = YAML.load(ERB.new(IO.read(path)).result))
-        result.deep_symbolize_keys
+        merge_config_by_environment(result.deep_symbolize_keys, ENV['RAILS_ENV'].to_sym)
```

Comment:
hi @matrinox 

Shoryuken has also a standalone version, which is not Rails dependent, so `RAILS_ENV` is not an option. 

If you want to make it environment based, I would recommend you to setup queue names via ENV variables.

```yml
  queues:
  - [<%= ENV['QUEUE1'] %>, 1]
  - [<%= ENV['QUEUE2'] %>, 2]
```

Reply:
Ahh, didn't know you could that. Thanks, I'll try it out